examples/monitoring-alert/README.md

```diff
@@ -0,0 +1,23 @@
+## Requirements
+
+No requirements.
+
```

Comment:
Provide a brief exec summary of what this example is?   Something like

Checks for secret deletion every 5 mins